main.tf

```diff
@@ -299,11 +299,31 @@ resource "aws_appautoscaling_policy" "autoscaling" {
 
   target_tracking_scaling_policy_configuration {
     predefined_metric_specification {
-      predefined_metric_type = "RDSReaderAverageCPUUtilization"
+      predefined_metric_type = "${var.predefined_metric_type_cpu}"
```

Comment:
RDSReaderAverageCPUUtilization and  RDSReaderAverageDatabaseConnections are the only metrics types you can use. I think it does not need a variable

Reply:
Thanks, I have removed the `predefined_metric_type_cpu` variable here:
https://github.com/claranet/terraform-aws-aurora/pull/38/commits/e424f3e25f6e799d0154ae13e73bd71a7bee68bd#diff-7a370d8342e7203b805911c92454f0f4R302